MapboxVisionTests/RecordCoordinatorTests.swift

```diff
@@ -72,6 +70,69 @@ final class RecordCoordinatorTests: XCTestCase {
         XCTAssert(directoryExists(at: recordingPath), "Recording should be saved at \(recordingPath) after recording is stopped")
     }
 
+    class TestClipRequest {
+        let start: Float
+        let end: Float
+
+        init(start: Float, end: Float) {
+            self.start = start
+            self.end = end
+        }
+    }
+
+    func testVideoClipping(withRequests requests: [TestClipRequest], _ message: String = "", line: UInt = #line) {
```

Comment:
Let's keep the naming convention (`test*`) for test-cases only.
```suggestion
    private func performVideoClipping(with requests: [TestClipRequest], _ message: String = "", line: UInt = #line) {
```
Also what's the point in `line` argument? Don't we want to know where exactly assert failed instead of where this function was called?

Reply:
It's a helper method, the test in which we interested is using it